frontend/tablet/src/components/Button.tsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react'
+import { StyleSheet, Text } from 'react-native'
+import { TouchableOpacity } from 'react-native-gesture-handler'
+
+interface Props {
+  title: string
+  onPress: () => void
+}
+
+const Button: React.FC<Props> = (props) => {
+  return (
+    <TouchableOpacity style={styles.container} onPress={props.onPress}>
+      <Text style={styles.text}>{props.title}</Text>
+    </TouchableOpacity>
+  )
+}
+
+export default Button
+
+const styles = StyleSheet.create({
+  container: {
+    backgroundColor: '#36DABC',
+    borderRadius: 12,
+    paddingHorizontal: 18,
+    paddingVertical: 12,
+  },
+  text: {
+    color: '#000000',
+    fontFamily: 'PT Sans Caption Bold',
+    fontSize: 18,
+    textTransform: 'uppercase',
+    textAlign: 'center',
```

Comment:
Extract all of the values into a shared `theme` file to ensure we stay consistent with the styles throughout the app. In this specific case, I mean colors, border radius, spacing - padding/margin, font sizes, font families.